app/main.py

```diff
@@ -1,6 +1,15 @@
-# write your imports here
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+from app.cinema.hall import CinemaHall
+from app.cinema.bar import CinemaBar
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list, hall_number: int, cleaner, movie: str):
```

Comment:
```suggestion
def cinema_visit(customers: list, hall_number: int, cleaner, movie: str):
```
why did you delete the type annotation?

Reply:
I made a mistake